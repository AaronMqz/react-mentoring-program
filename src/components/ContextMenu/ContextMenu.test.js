import React from "react";
import { render, screen, cleanup } from "@testing-library/react";
import ContextMenuComponent from "./ContextMenu";
import renderer from "react-test-renderer";

afterEach(cleanup);

describe("ContextMenu Tests:", () => {
  test("1. That component is visible.", () => {
    render(<ContextMenuComponent />);
    const text = screen.getByText("Edit");
    expect(text).toBeInTheDocument();
  });

  it("2. Snapshot renders correctly", () => {
    const tree = renderer.create(<ContextMenuComponent />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
