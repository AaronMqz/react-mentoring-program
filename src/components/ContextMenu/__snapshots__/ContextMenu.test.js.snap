// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContextMenu Tests: 2. Snapshot renders correctly 1`] = `
<div
  className="sc-beqWaB cqYDjH"
>
  <div
    className="sc-gueYoa HYHmj"
  >
    Edit
  </div>
  <div
    className="sc-gueYoa HYHmj"
  >
    Delete
  </div>
</div>
`;
